.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!--
+  This template is not mandatory.
+  It simply serves to provide a guide to allow a better review of pull requests.
+-->
+### What's is in this PR ?
```

Comment:
Suggestion: this section can be merged with `Relevant issues` (see example here: https://github.com/kuzzleio/kuzzle/pull/1093)